zebra/zebra_dplane.c

```diff
@@ -396,11 +396,6 @@ static enum zebra_dplane_result pw_update_internal(struct zebra_pw *pw,
 static enum zebra_dplane_result intf_addr_update_internal(
 	const struct interface *ifp, const struct connected *ifc,
 	enum dplane_op_e op);
-static enum zebra_dplane_result mac_update_internal(
```

Comment:
hmm - I'd like to do this a bit differently - a) it's confusing (imo) to expose an "internal" utility api, and b) the result is that we've gone from one copy of some shared code to two copies, which just doesn't feel good to me. I _think_ you want to be able to run the rmac-oriented initialization code on a ctx? Let's make that available independently from the common enqueue, stats pieces so we don't have to have two copies of those.

Reply:
> a) it's confusing (imo) to expose an "internal" utility api

Ops, yes... sorry about keeping the old function name, I don't know how to name it yet.

>  I think you want to be able to run the rmac-oriented initialization code on a ctx?

Yes.